git-pack/src/data/object.rs

```diff
@@ -4,6 +4,8 @@ use git_object::{BlobRef, CommitRef, CommitRefIter, ObjectRef, TagRef, TagRefIte
 
 use crate::data::Object;
 
+// FIXME: this entire thing should be in git-object! (probably) with
+// the possible exception of checksum checking
```

Comment:
I tentatively agree, let's please do this in a separate PR.

Reply:
OK, and I'll have to remove that comment as well, apparently I forgot to fix/remove it during my last cleanup pass. Sorry about that.